[All Hosts] Add articles about developing and publishing add-ins using VS Code. (#1457)
OfficeDev/office-js-docs-pr: docs/publish/publish-add-in-vs-code.md

---
title: Publish an add-in developed with Visual Studio Code

> description: Publish an add-in developed with Visual Studio Code

Reviewer: ```suggestion
description: How to publish an add-in developed with Visual Studio Code.
```

ms.date: 11/27/2019
localization_priority: Priority
---

# Publish an add-in developed with Visual Studio Code

> This article describes how to publish an Office Add-in that you created using the Yeoman generator and developed with [Visual Studio Code (VS Code)](https://code.visualstudio.com) or any other editor.

Reviewer: ```suggestion
You can publish an Office Add-in that you created using the Yeoman generator and developed with [Visual Studio Code (VS Code)](https://code.visualstudio.com) or any other editor.
```
